components/Container/Container.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { HTMLAttributes } from 'react';
+import { HTMLAttributes } from 'react';
 import { cnb } from 'cnbuilder';
 import {
   paddingTops,
   paddingBottoms,
   paddingVerticals,
   PaddingType,
-} from '../../utilities/datasource';
+} from '@/utilities/datasource';
```

Comment:
Nextjs introduce this way of easy import using the "@" - this means I'm importing form the utilities directory in the project root.